lib/constants.js

```diff
@@ -1,20 +1,15 @@
 'use babel'
 
-export const MachineStatusText = [
-  null,
-  'Offline',
-  'Online',
-  'Connected'
-]
+export const MachineStatusText = [null, 'Offline', 'Online', 'Connected']
 
 export const MachineStatus = {
   Offline: 1,
   Online: 2,
-  Connected: 3
+  Connected: 3,
```

Comment:
No it's not a syntax error. it's actually pretty common practice to leave it like this so that adding a new item into object creates only an insertion diff.